dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/ResInfoTest.java

```diff
@@ -14,13 +14,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.apache.dolphinscheduler.common.utils;
 
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Date;
+import org.apache.dolphinscheduler.common.model.Server;
+
+public class ResInfoTest {
+    @Test
+    public void testGetHeartBeatInfo() {
+        String info = ResInfo.getHeartBeatInfo(new Date());
+        Assert.assertEquals(info.split(",").length, 7);
```

Comment:
There is  a code some in Sonar analysis: Swap these 2 arguments so they are in the correct order: expected value, actual value.

https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&issues=AXAv9q6wCNVECePiTlLM&open=AXAv9q6wCNVECePiTlLM&pullRequest=1928

Reply:
fixed